bigframes/session/__init__.py

```diff
@@ -703,10 +703,8 @@ def _read_pandas_inline(
         try:
             local_block = blocks.Block.from_local(pandas_dataframe, self)
             inline_df = dataframe.DataFrame(local_block)
-        except pa.ArrowInvalid as e:
-            raise pa.ArrowInvalid(
-                f"Could not convert with a BigQuery type: `{e}`. "
-            ) from e
+        except pa.ArrowInvalid:
+            return None
```

Comment:
Please help to add comments here. For example, `An error is thrown for unsupported types, such as geo.`